Desktop Guide - WPF Initial Release (#13780)
dotnet/docs: docs/desktop-wpf/migration/differences-from-net-framework.md

## NuGet package references

If your .NET Framework app lists its NuGet dependencies in a *packages.config* file, migrate to the [`<PackageReference>`](/nuget/consume-packages/package-references-in-project-files) format.

01. In Visual Studio, find the *Solution Explorer* pane.
01. With your WPF app, right-click **packages.config** > **Migrate packages.config to PackageReference**.

![Upgrading to PackageReference](media/differences-from-net-framework/package-reference-migration.png)

A dialog will appear showing calculated top-level NuGet dependencies and asking which other NuGet packages should be promoted to top level. Click **Ok** and the *packages.config* file will be removed from the project and `<PackageReference>` elements will be added to the project file.

When your project uses `<PackageReference>`, packages aren't stored locally in a *packages* folder, they're stored globally. Open the project file and remove any `<Analyzer>` elements that referred to the *packages* folder. These analyzers are automatically included with the NuGet package references.

## Code Access Security (CAS)

> WPF no longer supports Code Access Security (CAS) as .NET Core doesn't support it. All CAS-related functionality is treated under the assumption of full-trust. Therefore, WPF has been removing CAS-related code. The public API surface of these types still exists to ensure that calls into these types succeed. However, these calls don't actually do anything.

Reviewer: ```suggestion
Code Access Security (CAS) is not supported by .NET Core or WPF for .NET Core. All CAS-related functionality is treated under the assumption of full-trust. Therefore, WPF has been removing CAS-related code. The public API surface of these types still exists to ensure that calls into these types succeed. However, these calls don't actually do anything.
```
